server/showcase.go

```diff
@@ -15,40 +15,47 @@
 package server
 
 import (
+	"context"
 	"fmt"
 	"io"
 	"strconv"
 	"strings"
 	"sync"
+	"sync/atomic"
 	"time"
 
 	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/empty"
 	pb "github.com/googleapis/gapic-showcase/server/genproto"
-	"github.com/grpc/grpc-go/status"
-
-	"golang.org/x/net/context"
-
 	lropb "google.golang.org/genproto/googleapis/longrunning"
 	statuspb "google.golang.org/genproto/googleapis/rpc/status"
 	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
+type uniqID struct {
+	i int64
+}
+
+func (u *uniqID) id() int64 {
+	return atomic.AddInt64(&u.i, 1) - 1
```

Comment:
Using current time as ID can lead to weird failures during clock resets and concurrent calls. Unlikely to cause problem since we're using it to test, but thought we might as well harden against it.